### ISSUE TITLE
how i can show blink icon or marker on the area part?

### ISSUE THREAD

**Issue by pichlou** [etienne-martin/Mapify#29]

**Comment by etienne-martin** [comment]
Can you provide a more detailed explanation of what you're trying to do?

**Comment by pichlou** [comment]
yes,i have a user tracking project in a place such as hospital,i mapping the floor plan image with area tag and want to show user location in hospital floor plan with an icon,but i don't know how!

**Comment by etienne-martin** [comment]
I fear that this is not possible with the current version of this library. You'll have to implement this functionality yourself.

**Comment by adeiry** [comment]
@pichlou Did you figure it out?
